rs/recovery/src/app_subnet_recovery.rs

```diff
@@ -38,7 +38,7 @@ pub enum StepType {
     BlessVersion,
     /// This step issues an ic-admin command that will create an upgrade proposal for the troubled subnet. Note that the subnet nodes will only upgrade after we proposed the corresponding recovery CUP referencing the new registry version.
     UpgradeVersion,
-    /// Now we are ready to restart the subnet's computation. In order to do that, we need to instruct the subnet to start the computation from a specific height and state with a specific hash. We can only do this by writing a special message for the subnet into the registry. This step generates an ic-admin command creating a proposal with such an instruction for the subnet containing the hash of the state we obtained in the previous step and with a height strictly higher that the latest finalized height. Potentially, if we want to recover the subnet on a new set of nodes, their IDs can be specified as well. If the subnet has an ECDSA key, we also need to specify a backup subnet to reshare the key from.
+    /// Now we are ready to restart the subnet's computation. In order to do that, we need to instruct the subnet to start the computation from a specific height and state with a specific hash. We can only do this by writing a special message for the subnet into the registry. This step generates an ic-admin command creating a proposal with such an instruction for the subnet containing the hash of the state we obtained in the previous step and with a height strictly higher that the latest finalized height. Potentially, if we want to recover the subnet on a new set of nodes, their IDs can be specified as well. If the subnet has any Chain keys, we also need to specify a backup subnet to reshare the key from.
```

Comment:
Just a quick general question: Is there a reason these long comments are in one line?

Reply:
Turns out there wasn't, fixed!

Reply:
Awesome!